### PR TITLE
#2570 - Add Bank transfer fee tooltip

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx [hunk 1]
```diff
@@ -157,10 +158,27 @@ const BankTransferRedeemLayout: React.FunctionComponent<IProps> = ({
...
+                        new BigNumber(bankFee).mul(100), // to get percentage value we have to multiply ba 100
+                        ECurrency.EUR,
+                        EMoneyFormat.ULPS,
+                        false,
+                        ERoundingMode.HALF_UP,
+                      ),
+                    }}
+                  />
+                }
+                preventDefault={false}
```
Comment (desfero) [inline app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: Do you need prevent default here?

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx [hunk 2]
```diff
@@ -157,10 +158,27 @@ const BankTransferRedeemLayout: React.FunctionComponent<IProps> = ({
...
-              <Heading level={3} decorator={false} size={EHeadingSize.SMALL}>
-                <FormattedMessage id="bank-transfer.redeem.init.redeem-fee" />
-              </Heading>
-
+              <Tooltip
+                content={
+                  <FormattedMessage
+                    id="bank-transfer.redeem.init.redeem-fee.tooltip"
+                    values={{
+                      fee: getFormattedMoney(
```
Comment (desfero) [inline app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: can we use our `Money` component here?

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx [hunk 3]
```diff
@@ -157,10 +158,27 @@ const BankTransferRedeemLayout: React.FunctionComponent<IProps> = ({
...
+                        new BigNumber(bankFee).mul(100), // to get percentage value we have to multiply ba 100
```
Comment (desfero) [inline app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: hm, what's the real reason to multiply? It's not clear from the comment in this case. As if you see the message it says `{fee}%` so what's the reason to use `getFormattedMoney`?

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx (hunk 2 above)
Comment (drakkein) [reply app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: It's not working. Element is not added to translated string :(

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx (hunk 1 above)
Comment (drakkein) [reply app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: Nope :D

File: app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx (hunk 3 above)
Comment (drakkein) [reply app/components/modals/tx-sender/redeem/BankTransferRedeemInit.unsafe.tsx]: Added more detailed comment

**Pull request merged**